### ISSUE TITLE
Process to update vendored code.

### ISSUE THREAD

**Issue by Carreau** [napari/napari#3267]
From #3265

My understanding is that the process to update vendored files is manual.
We might want to have a bit of automation, and verification that no manual changes are put in those files that are not upstream.

I'm not too sure how the files are cherry-picked, and/or if modifications are applied to them (like imports).

**Comment by sofroniewn** [comment]
Yeah, good to open a discussion on this - i think there is room for us to improve our behaviour
